releasenotes/releasenotes_v5.rst

```diff
@@ -5,23 +5,91 @@ Current Firely Server release notes (v5.x)
+
+Existing installations may be affected by the fixes on Composite search parameters for the SQL Server database repository.
+
+Database
+^^^^^^^^
+
+* The SQL Server database schema is upgraded from version 26 to 27. The upgrade will be applied automatically, but if you have a very large database you may want to apply it manually using the script FS_SchemaUpgrade_Data_v26_v27.
+* This implies that you also need to upgrade Firely Server Ingest to version 2.2.0, to match the new database schema.
+
+Configuration
+^^^^^^^^^^^^^
+
+* The ``HistoryOptions`` are removed as configuration option, so you can remove it from your configuration in ``appsettings.instance.json`` or environment variables as well. The returned resources will be limited by the settings in the ``BundleOptions``, see :ref:`bundle_options`.
+* The Bulk Data Export upgrades (see below) come with a few extra configuration settings, see :ref:`feature_bulkdataexport`
+
+Features
+^^^^^^^^
+* Firely Server is upgraded to the release version (5.0.0) of FHIR R5. If you have your administration database in SQL Server or MongoDB, this means that the conformance resources will be :ref:`re-imported <conformance_import>`.
+* We included ``errataR5.zip`` with fixes to a few resources and search parameters that have errors in the specification. These are imported automatically at startup.
+* We upgraded Firely Server to the latest SDK 5.1.0, see its `releasenotes <https://github.com/FirelyTeam/firely-net-sdk/releases/tag/v5.1.0>`_.
+* Support for §170.315(b)(10) - Electronic Health Information Export, as required by the end of 2023.
+* Bulk Data Export is upgraded to BDE 2.0, with support for: //TODO: Check further
+  
+  * patient Filter
+  * _elements filter
+  * HTTP POST with a Parameters resource
+  * export to Azure Blob or Azure Files //TODO: Add settings to bulkdateexport.rst (and maybe appsettings.rst) after merge of develop
+
+* Our public Postman collection proving support for US-Core is updated, see :ref:`compliance_uscore`
+* Updated our vulnerability scanning, to further enhance your trust in our binaries and images.
+* Cross-origin requests (CORS) are restricted to requests from secure connections.
+* The following security headers were added:
+
+  * to the html output (the homepage): ``script nonce="..."``, ``cache-control``, ``content-security-policy``, ``referrer-policy``, ``x-content-type-options``
+  * and to API response: ``cache-control:no-store``
+
+* You can configure limits on Kestrel, see :ref:`hosting_options`, although using a :ref:`reverse proxy<deploy_reverseProxy>` is still preferred.
+* Added a configuration error to the log if the default informationmodel (aka FHIR version) is not loaded in the pipeline.
+
 Fix
 ^^^
+* Composite search parameters are more accurately supported on SQL Server. Previously a match could be made across components, but not anymore. This comes with a database migration, see above.
```

Comment:
Can you explain a bit more on detail that the old behavior was wrong and the new behavior is what you actually want?

Reply:
Done.